components/script/dom/webidls/OffscreenCanvas.webidl

```diff
@@ -13,7 +13,7 @@ dictionary ImageEncodeOptions {
 
 //enum OffscreenRenderingContextId { "2d", "webgl", "webgl2" };
 
-[Exposed=(Window,Worker)/*, Transferable*/, Pref="dom_offscreen_canvas_enabled"]
+[Exposed=(Window,Worker)/*, Transferable*/, Pref="dom_canvas_offscreen_enabled"]
```

Comment:
I'm not thrilled by this rename, since the current pref contain the actual interface name that is enabled, which follows the model of our other dom preferences.

Reply:
I'm happy to undo this particular rename.